docs/content/development/extensions-core/kafka-ingestion.md

```diff
@@ -339,12 +339,10 @@ keeping the number of segments low are:
 In many production installations which have been ingesting events for a long period of time, these suggestions alone
 will not be sufficient to keep the number of segments at an optimal level. It is recommended that scheduled re-indexing
 tasks be run to merge segments together into new segments of an ideal size (in the range of ~500-700 MB per segment).
-Currently, the recommended way of doing this is by running periodic Hadoop batch ingestion jobs and using a `dataSource`
-inputSpec to read from the segments generated by the Kafka indexing tasks. Details on how to do this can be found under
-['Updating Existing Data'](../../ingestion/update-existing-data.html). Note that the Merge Task and Append Task described
-[here](../../ingestion/tasks.html) will not work as they require unsharded segments while Kafka indexing tasks always
-generated sharded segments.
-
-There is ongoing work to support automatic segment compaction of sharded segments as well as compaction not requiring
-Hadoop (see [here](https://github.com/druid-io/druid/pull/1998) and [here](https://github.com/druid-io/druid/pull/3611)
-for related PRs).
+Currently, the recommended way of doing this is turning on the [automatic compaction of coordinators](../../design/coordinator.html#compacting-segments).
```

Comment:
not sure if we should recommend one over the other at this time when new feature hasn't been used much and comes with cautionary notes. I think we should simply describe both options and let users choose whatever is convenient to them.
so far, I like batch re-indexing as it is relatively simpler to understand for users already using batch indexing and works for all cases.

makes sense ?

Reply:
Sounds make sense. Changed as suggested.